src/runtime/types.cpp

```diff
@@ -4405,7 +4403,7 @@ void setupRuntime() {
     type_cls->giveAttr(
         "__new__", new BoxedFunction(BoxedCode::create((void*)typeNewGeneric, UNKNOWN, 4, false, false, "type.__new__"),
                                      { NULL, NULL }));
-    type_cls->giveAttr("__repr__", new BoxedFunction(BoxedCode::create((void*)typeRepr, STR, 1, "type.__repr__")));
```

Comment:
What code relies on the type of `type.__repr__`?  This change is a bit scary because it could affect a lot of things.